libs/features/quest/src/quest-template-locale/quest-template-locale.integration.spec.ts

```diff
@@ -0,0 +1,198 @@
+import { QuestTemplateLocale } from '@keira/shared/acore-world-model';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { MysqlQueryService } from '@keira/shared/db-layer';
+import { of } from 'rxjs';
+import { QuestPreviewService } from '../quest-preview/quest-preview.service';
+import { QuestHandlerService } from '../quest-handler.service';
+import { QuestTemplateLocaleComponent } from './quest-template-locale.component';
+import { MultiRowEditorPageObject, TranslateTestingModule } from '@keira/shared/test-utils';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ToastrModule } from 'ngx-toastr';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { RouterTestingModule } from '@angular/router/testing';
+import { KEIRA_APP_CONFIG_TOKEN, KEIRA_MOCK_CONFIG } from '@keira/shared/config';
+import Spy = jasmine.Spy;
+
+class QuestTemplateLocalePage extends MultiRowEditorPageObject<QuestTemplateLocaleComponent> { }
+
+describe('QuestTemplateLocale integration tests', () => {
+  const id = 1234;
+  let fixture: ComponentFixture<QuestTemplateLocaleComponent>;
+  let queryService: MysqlQueryService;
+  let querySpy: Spy;
+  let handlerService: QuestHandlerService;
+  let page: QuestTemplateLocalePage;
```

Comment:
Please make sure to not use shared variables. 
See https://francescoborzi.github.io/ngx-page-object-model/docs/best-practices/avoid-shared-variables
It would be better to avoid having shared non-immutable variables across different tests and just initialise and return them in a setup function. 
For reference see:
https://github.com/azerothcore/Keira3/pull/3227/files